packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -0,0 +1,67 @@
+import { attr, observable } from '@microsoft/fast-element';
+import { DesignSystem, FoundationElement } from '@microsoft/fast-foundation';
+import { template } from '../theme-provider/template';
+import { styles } from './styles';
+import {
+    WaferMapColorsScale,
+    WaferMapDie,
+    WaferMapOrientation,
+    WaferMapQuadrant
+} from './types';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-wafer-map': WaferMap;
+    }
+}
+
+/**
+ * A nimble-styled WaferMap
+ */
+export class WaferMap extends FoundationElement {
+    @attr({
+        attribute: 'quadrant'
+    })
+    public quadrant: WaferMapQuadrant = WaferMapQuadrant.topLeft;
+
+    @attr({
+        attribute: 'orientation'
+    })
+    public orientation: WaferMapOrientation = WaferMapOrientation.top;
+
+    @attr({
+        attribute: 'max-characters'
+    })
+    public maxCharacters = 4;
+
+    @attr({
+        attribute: 'show-die-labels'
+    })
+    public showDieLabels = true;
+
+    @attr({
+        attribute: 'die-labels-suffix'
+    })
+    public dieLabelsSuffix = '';
+
+    @attr({
+        attribute: 'is-continuous'
+    })
+    public isContinuous = true;
```

Comment:
As mentioned above all boolean properties MUST default to false.

Should we consider making this property an enum instead? Some benefits:
1. Maybe there would be additional configuration modes in the future?
2. For users of the control, it may not be intuitive that `continuous === false` means categorical. With an enum configuration it is much more explicit about what the options are.

Assuming this would make sense as an enum the following is one example to consider:

```ts
    //types.ts
    export const WaferMapColorsScaleMode = {
        continuous: 'continuous',
        categorical: 'categorical'
    } as const;
    
    export type WaferMapColorsScaleMode =
        typeof WaferMapColorsScaleMode [keyof typeof WaferMapColorsScaleMode ];

    // wafer-map/index.ts
    @attr({
        attribute: 'colors-scale-mode'
    })
    public colorsScaleMode = WaferMapColorsScaleMode.continuous;
```